sql/src/test/java/io/crate/integrationtests/MetricsITest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.integrationtests;
+
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+public class MetricsITest extends TransportSQLActionTest {
+
+    @Test
+    public void testSimpleQueryOnMetrics() {
+        execute("select 1");
+
+        execute("select node['name'], min, max, percentiles['90'] from sys.jobs_metrics order by 3 desc");
```

Comment:
shouldn't we query for all columns here to test their expressions as well? otherwise, e.g. removing the `mean` column would not break anything...